src/main/fc/settings.c

```diff
@@ -713,6 +715,7 @@ const clivalue_t valueTable[] = {
 #ifdef USE_ESC_SENSOR
     { "esc_sensor_halfduplex",          VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_ESC_SENSOR_CONFIG, offsetof(escSensorConfig_t, halfDuplex) },
 #endif
+    { "led_inversion",                  VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 255 }, PG_STATUS_LED_CONFIG, offsetof(statusLedConfig_t, inversion) },
```

Comment:
I have got my code for CLI array assignments almost ready for a PR (maybe later on tonight). Once that is in, we could do `set led_inversion = 0, 1, 0` and store it in a `uint8_t ledInversion[STATUS_LED_NUMBER]`.

The other question is, do we even need to have per LED inversion? In the existing targets, either all LEDs are inverted, or none.